pro.vpup.vpuppr.yml

```diff
@@ -0,0 +1,46 @@
+id: pro.vpup.vpuppr
+runtime: org.freedesktop.Platform
+runtime-version: "22.08"
+sdk: org.freedesktop.Sdk
+command: vpuppr.x86_64
+finish-args:
+    - "--socket=x11"
+    - "--share=ipc"
+    - "--share=network" # Required for communication between openseeface and vpuppr
+    - "--device=all" # webcam
+    - "--socket=pulseaudio"
+    - "--filesystem=home" # to see all files (vpuppr uses its own file picker), and function to select models properly
+modules:
+
+  # --- Puppeteer ---
+  - name: puppeteer
+    buildsystem: simple
+    build-commands:
+      - mkdir /app/bin
+      - mkdir -p /app/share/icons/hicolor/scalable/apps
+      - mkdir -p /app/share/{pro.vpup.vpuppr, vpuppr}/resources/
+      - install -Dm644 pro.vpup.vpuppr.metainfo.xml -t /app/share/metainfo/
+
+      - install -Dm 644 -t /app/share/applications pro.vpup.vpuppr.desktop
+      - mv pro.vpup.vpuppr-scalable.svg /app/share/icons/hicolor/scalable/apps/pro.vpup.vpuppr.svg
+
+      - mv vpuppr.* /app/bin/
+      - chmod +x /app/bin/vpuppr.x86_64
+      - mv resources /app/share/pro.vpup.vpuppr/
+
+
+      - ln -s /app/share/vpuppr/ /app/bin/resources
+      - ln -s /app/share/pro.vpup.vpuppr /app/share/vpuppr
+#      - ls /app/share/vpuppr/resources/extensions/openseeface-tracker
+#      - chmod +x /app/share/pro.vpup.vpuppr/resources/extensions/openseeface-tracker/OpenSeeFace
+
+    sources:
+      - type: archive
+        url: https://github.com/virtual-puppet-project/vpuppr/releases/download/0.9.0/vpuppr_0.9.0_linux_flatpak_compat.zip
```

Comment:
we have a Godo base App that would allow building from source... (and possibly on aarh64)

Reply:
I'd rather pull binaries from upstream, since upstream provides more than just Godot binaries (they have an extensions system too, which I would also need to build); I've discussed building aarch64 binaries there, when they provide binaries for it I'll enable support.

Reply:
Also @hfiguiere, https://github.com/flatpak/flatpak-builder-tools/issues/310 would need to be resolved before I could build from source.